tests/src/sizeondisk/sodbench/SoDBench.cs

```diff
@@ -370,7 +370,7 @@ private static SizeReportingNode GetDeploymentSize()
                     ProcessStartInfo dotnetPublish = new ProcessStartInfo()
                     {
                         FileName = s_dotnetExe.FullName,
-                        Arguments = $"publish -c Release --runtime {os} --output {publishDir.FullName}", // "out" is an arbitrary project name
+                        Arguments = $"publish -c Release --runtime {os} --output {publishDir.FullName} /p:UseSharedCompilation=false",
```

Comment:
Could you please add a comment on what the `UseSharedCompilation=false` do? Optionally, it seems that this parameter can be set as an environment variable too.

Reply:
fixed